components/org.wso2.carbon.security.caas/src/main/java/org/wso2/carbon/security/caas/user/core/store/connector/AuthorizationStoreConnector.java

```diff
@@ -153,37 +166,41 @@ void init(String storeId, AuthorizationStoreConnectorConfig authorizationStoreCo
 
     /**
      * Add new resource.
+     *
      * @param resourceNamespace Namespace of the resource.
-     * @param resourceId Id of the resource.
-     * @param userId User id of the owner.
-     * @param userDomainId Id of the user's domain.
+     * @param resourceId        Id of the resource.
+     * @param userId            User id of the owner.
+     * @param userDomainName    Name of the user's domain.
      * @return New Resource.
      * @throws AuthorizationStoreException
      */
-    Resource addResource(String resourceNamespace, String resourceId, String userId, String userDomainId)
+    Resource addResource(String resourceNamespace, String resourceId, String userId, String userDomainName)
```

Comment:
Yes, you are correct. This change is to simply remove the Domain Id and replace it with Domain name. Will remove the Domain parameter in the next refactoring commit.